site/app/views/UserProfileView.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace app\views;
+
+use app\libraries\DateUtils;
+use app\models\User;
+
+/**
+ * Class UserProfileView
+ * @package app\views
+ */
+class UserProfileView extends AbstractView {
+    /**
+     * @param User $user
+     * @param string $change_name_text
+     * @param bool $database_authentication
+     * @param string $csrf_token
+     * @return string
+     */
+    public function showUserProfile(
+        User $user,
+        string $change_name_text,
+        bool $database_authentication,
+        string $csrf_token
+    ) {
```

Comment:
I'm not sure how I feel about splitting this onto multiple lines.  It does increase readability but it doesn't seem very consistent with the rest of the code base.  I would like to solicit additional feedback from @bmcutler and @MasterOdin .